File: 프로그래머스/lv2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.js
function solution(s) {
    var answer = '';
    let array = s.split(" ");
    for (let i = 0; i < array.length; i++)
        {
            if(array[i].length === 0) 
            {
                answer += " ";
                continue;
            }
            let word = array[i];
            word = word.toLowerCase();
            let first_char = word[0].toUpperCase();
            let others = word.slice(1).toLowerCase();
            word = first_char + others;
            if (i == array.length-1)
                {
                    answer += word;
                }
            else answer +=word +" ";
        }
    if( answer.length > s.length)  answer = answer.slice(0,-1);
    return answer;
}

File: 프로그래머스/lv2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string s) {
    string answer = "";
    if (s[0] >= 97 && s[0] <=122) 
    {
        s[0] -= 32;
    }
    for ( int i = 1; i < s.size(); i ++){
        if (s[i] >=65 && s[i]<= 90) 
        {
            s[i] +=32;
        }
        if ( s[i-1] == ' ')
        {
            if ( s[i] >= 97 && s[i] <= 122) 
            {
                s[i] -= 32;
            }
        }
        
    }
    answer = s ;
    return answer;
}